src/smoosh_channel.erl

```diff
@@ -156,7 +156,8 @@ handle_info(check_window, State0) ->
     {ok, State} = code_change(nil, State0, nil),
     #state{paused = Paused, name = Name} = State,
     StrictWindow = smoosh_utils:get(Name, "strict_window", "false"),
-    FinalState = case {not Paused, smoosh_utils:in_allowed_window(Name)} of
+    {_, {HH, MM, _}} = calendar:universal_time(),
+    FinalState = case {not Paused, smoosh_utils:in_allowed_window({HH, MM}, Name)} of
```

Comment:
This is just a very minor downside of the approach (feel free to ignore, I like current solution and don't have strong opinion against it).
This approach forces test suite to know about format of the argument. We could pass result of `calendar:universal_time/0` as is (universal_time is standard erlang format and we don't need `hh_mm(date_time)` in test suite) and do:
```
in_allowed_window({_, {HH, MM, _}}, To) ->
    ....
```

Reply:
@iilyak that does sound like a good idea, I like that too

Reply:
I like the idea of passing the entire term returned by `calendar:universal_time/0`, but I think the tests would still need some massaging to convert
```elixir
iex(1)> DateTime.utc_now()
#DateTime<2019-12-13 22:18:14.579614Z>
```
into
```erlang
(node1@127.0.0.1)17> calendar:universal_time().
{{2019,12,13},{22,17,37}}
```
right?